scripts/nrf_provision/fast_pair/README.rst

```diff
@@ -10,7 +10,7 @@ Fast Pair provision script
 This Python script generates a hexadecimal file required for the :ref:`Google Fast Pair integration in the nRF Connect SDK <ug_bt_fast_pair>`.
 
 .. note::
-   When you :ref:`use Google Fast Pair with the nRF Connect SDK <ug_bt_fast_pair>`, the script is automatically called by the build system.
+   When you :ref:`use Google Fast Pair with the nRF Connect SDK <ug_bt_fast_pair>` and enable the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option, the script is automatically called by the build system.
```

Comment:
Maybe we should also mention that if you do not use sysbuild and enable CONFIG_BT_FAST_PAIR, the script is also automatically called by the build system.